### DISCUSSION TITLE
React Native Axios vs Postman

### DISCUSSION THREAD

**Discussion by ThushalIntervest** [axios/axios#5304] [Q&A]
I have experienced API Calling using React Native AXIOS getting more Response Time than Postman. 

Can someone Please Explain why that is happening? ( I know POSTMAN always check CORS and one is GET, POST. Rather than this, ANY OTHER reasons for my question.)

**Comment by DevinByteX** [comment]
I'm also having same issue. Did you find any solution for this.

**Comment by jasonsaayman** [comment]
please check by searching the issues for anything you may need to change as react native has a couple issues with axios
